eql/functions.py

```diff
@@ -319,6 +319,29 @@ def run(cls, source, substring):
             return substring.lower() in source.lower()
         return False
 
+@register
+class Strip(FunctionSignature):
```

Comment:
This looks straightforward and good.
Can you add a test for each function `lstrip`/`rstrip`/`strip` after splitting?

The queries and expected results are here:
https://github.com/endgameinc/eql/blob/master/eql/etc/test_queries.toml#L1-L3

And the source data is here:
https://github.com/endgameinc/eql/blob/master/eql/etc/test_data.json

From there, you can run the local tests by running `make test` on Linux/Mac. On windows, you can run
```console
$ python setup.py install
$ python setup.py install eql[test]
$ python setup.py -q lint
$ python setup.py -q test
```

Reply:
Resolved in eca2846